reference/commands/list.rst

```diff
@@ -26,6 +26,11 @@ conan list
                             Remote names. Accepts wildcards
       -c, --cache           Search in the local cache
 
+.. warning::
+
+    In some circumstances, `--format html` might produce semantically different results from `--format json`, like
+    showing a different amount of packages (there is a bug `GitHub issue #13022 <https://github.com/conan-io/conan/issues/13022>`_). Use it on your own risk.
```

Comment:
I think it would be more accurate to say that is not implemented yet in <=2.0-beta9 and then remove the warning for the next release, because at least a very basic output will be there. I think it's a bit misleading to say that it may produce different results when the reality is that is not working at all because it has not been implemented yet.

Reply:
well, it's kinda working, as it produces an output file, and it's valid .html file that could be opened. e.g. there is no error message, crash, etc. it just doesn't include any data.
that's why it's so confusing, and I'd preferred if it just raised an error immediately for any `--format html` input instead.

Reply:
What about something similar to this?

```
.. warning::

    The html formatter (--format=html) is not fully implemented yet, so it will create an empty html file
    with no packages information. This is `a reported bug
    <https://github.com/conan-io/conan/issues/13022>`_ that happens in Conan <= 2.0-beta9
    and is planned to be solved in 2.0-beta10
```

Besides that, this should be pointed to the _release/2.0.0-beta_ branch to be published in the docs or it will never be published if we fix it in the next release

Reply:
okay, I've changed a message and base.